addons/web/static/src/scss/bootstrap_review_backend.scss

```diff
@@ -192,6 +192,10 @@ $-o-theme-text-colors: null;
     @extend %-bg-black;
 }
 
+.bg-inherit {
```

Comment:
This is not related to white/black classes (even if you slightly modified the note).
I propose to move it in the " _Odoo custom text/bg colors classes generation_" section, in its own point (n6 then), around L269

Reply:
Understood, I placed it here in regards to the "bg-transparent" utility which felt similar to me but indeed behaves differently since it extends the bg-black rather than applying a `background-color: transparent`.